src/rgw/rgw_asio_frontend.cc

```diff
@@ -522,6 +537,8 @@ void AsioFrontend::accept(Listener& l, boost::system::error_code ec)
     throw ec;
   }
   auto socket = std::move(l.socket);
+  boost::asio::ip::tcp::no_delay options(l.use_nodelay);
```

Comment:
nit: there's an alias for `using tcp = boost::asio::ip::tcp;` above, so this can just be `tcp::no_delay`

Reply:
Done